lib/graphql/dig.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+module GraphQL
+  module Dig
+    # implemented using the old activesupport #dig instead of the ruby built-in
+    # so we can use some of the magic in Schema::InputObject and Query::Arguments
+    # to handle stringified/symbolized keys.
+    #
+    # @param args [Array<[String, Symbol>] Retrieves the value object corresponding to the each key objects repeatedly
+    # @return [Object]
+    def dig(*args)
+      val = self[args.shift]
+      if val.nil? || (args.respond_to?(:empty?) ? !!args.empty? : !args)
```

Comment:
Won't `args` always be an array here? Why should we check `args.respond_to?(:empty?)`?

Reply:
you're right - i just copied the code from:

https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/object/blank.rb#L18

i'll update it.